pages/1.13/cli/command-reference/dcos-cluster/dcos-cluster-setup/index.md

```diff
@@ -3,63 +3,51 @@ layout: layout.pug
 navigationTitle:  dcos cluster setup
 title: dcos cluster setup
 menuWeight: 6
-excerpt: Configuring a connection to a DC/OS cluster
-
+excerpt: Configuring the CLI to communicate with a cluster
 enterprise: false
 ---
 
 # Description
-
-The `dcos cluster setup` command will configure the connection to a DC/OS cluster, authenticate to DC/OS, and attach to the cluster.
-
-It will also automatically install the [Core and Enterprise CLI plugins](/1.13/cli/plugins/).
+The `dcos cluster setup` command will configure the CLI to communicate with a cluster.
```

Comment:
Why did we remove the note stating that the enterprise CLI is automatically installed? This is pretty new behavior and people might get confused and still install it.